arviz/data/io_cmdstanpy.py

```diff
@@ -680,8 +726,12 @@ def from_cmdstanpy(
         Constant data used in the sampling.
     predictions_constant_data : dict
         Constant data for predictions used in the sampling.
-    log_likelihood : str, list of str
-        Pointwise log_likelihood for the data.
+    log_likelihood : str, list of str, dict of {str: str}
+        Pointwise log_likelihood for the data. If a dict, its keys should represent var_names
+        from the corresponding observed data and its values the stan variable where the
+        data is stored. By default, if a variable ``log_lik`` is present in the Stan model,
+        it will be retrieved as pointwise log likelihood values. Use ``False`` to avoid this
+        behaviour.
```

Comment:
@mitzimorris @ahartikainen does this sound like the desired behaviour to have sensible defaults? I think it was Mitzi who said that stan packages in the R world had this convention but my memory could be playing tricks on me

Reply:
Sounds ok.